packages/account-tree-controller/src/AccountTreeController.ts

```diff
@@ -723,6 +746,11 @@ export class AccountTreeController extends BaseController<
     this.update((state) => {
       state.accountTree.selectedAccountGroup = groupId;
     });
+    this.messagingSystem.publish(
+      `${controllerName}:selectedAccountGroupChange`,
+      groupId,
+      currentSelectedGroup,
```

Comment:
Would you be ok to rename `currentSelectedGroup` to `previousSelectedGroup` for consistency with other methods?